etna/pipeline/assemble_pipelines.py

```diff
@@ -0,0 +1,62 @@
+from typing import Any
+from typing import List
+from typing import Optional
+from typing import Sequence
+from typing import Union
+
+from etna.models.base import BaseModel
+from etna.pipeline.pipeline import Pipeline
+from etna.transforms import Transform
+
+
+def assemble_pipelines(
+    models: Union[BaseModel, Sequence[BaseModel]],
+    transforms: Sequence[Union[Transform, Sequence[Optional[Transform]]]],
+    horizons: Union[int, Sequence[int]],
+) -> List[Pipeline]:
+    """Create pipelines from input horizons and models or sequence of models.
+
+    Parameters
+    ----------
+    models:
+        Instance or Sequence of the etna Model
+    transforms:
+        Sequence of the transforms
+    horizons:
+        Sequence of horizons
+
+    Raises
+    ------
+    ValueError:
+        If the length of models sequence not equals to length of horizons sequence.
+    Returns
+    -------
+    list of pipelines
+    """
+    n_models = len(models) if isinstance(models, Sequence) else 1
+    n_horizons = len(horizons) if isinstance(horizons, Sequence) else 1
+    n_transforms = 1
+
+    for transform in transforms:
```

Comment:
This naming considered bad practice. Since it is hard to tell `transform` and `transforms` apart.